### PR TITLE
modify README and hbase-sql since the name of JAR has changed

### PR DESCRIPTION
**Pull request by Zhangshunyu**
Since the name of JAR has been changed in POM.XML, we have to Synchronize changes to other files.

### CODE REVIEW

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by Zhangshunyu** [comment]
test this please

**Comment by eason-lyx** [comment]
yes ,i test this.it is ok.
hbase-sql need to use the new jar package(astro-project-0.10.0.jar),not spark-sql-on-hbase-1.0.0.jar.

**Comment by ravipesala** [comment]
The version number got changed to 0.1.0, so please update as per that.

**Comment by ravipesala** [comment]
ok to test

File: bin/hbase-sql [hunk 1]
```diff
@@ -33,7 +33,7 @@ fi
 
 export FWDIR=$SPARK_HOME
 export ASTRO_SOURCE="$(cd "`dirname "$0"`"/..; pwd)"
-export JAR="$ASTRO_SOURCE/target/spark-sql-on-hbase-1.0.0.jar"
+export JAR="$ASTRO_SOURCE/target/astro-project-0.10.0.jar"
```
Comment (WangTaoTheTonic) [inline bin/hbase-sql]: can we add this jar into classpath with format of regex in case that the release version will change?

**Comment by Zhangshunyu** [comment]
@ravipesala  Update has been completed.

**Comment by ravipesala** [comment]
retest this please

**Comment by Zhangshunyu** [comment]
@ravipesala  Test what? Nothing changed but README...

**Comment by ravipesala** [comment]
@Zhangshunyu  I am asking Jenkins to test, not you. And also @WangTaoTheTonic review comment is valid.Please consider it.

**Comment by Zhangshunyu** [comment]
The Astro v0.1.0 you released 3 days ago had sth wrong with it‘s boot script "bin/hbase-sql", you'd better merge this PR first to avoid the mistake.

**Comment by ravipesala** [comment]
@Zhangshunyu  if you are facing any problem in fixing the review comment, please let me know. I can help you. But this is not the proper way of communicating in community. Any way I am merging this PR.

**Comment by chenliang613** [comment]
@Zhangshunyu  I agree ravipesala's comments, it would be better if you could please pay attention to the communication words.

**Comment by Zhangshunyu** [comment]
@ravipesala @chenliang613 I feel sorry for my tone of voice, and I will pay attention in the future. 
I am very sincere to say sorry to you,thanks for your understanding.

**Pull request merged**